cmd/clean.go

```diff
@@ -17,7 +19,14 @@ func CleanCommand(repo *core.Repo, gh func(context.Context) core.Gh) *cli.Comman
 		Action: func(cCtx *cli.Context) error {
 			repo.Fetch(cCtx.Context)
 			localPrs := repo.AllPrs(cCtx.Context)
-			for _, pr := range localPrs {
+
+			type cleanResult struct {
+				err error
+				pr  core.LocalPr
+			}
+
+			cleanPr := func(wg *sync.WaitGroup, out chan cleanResult, pr core.LocalPr) {
+				defer wg.Done()
```

Comment:
here you could acquire a semaphore so that we limit how many pull request we fetch in //